contracts/VaultFactory.vy

```diff
@@ -95,19 +125,44 @@ def api_version() -> String[28]:
     """
     return API_VERSION
 
+@view
+@external
+def protocol_fee_config() -> PFConfig:
+    """
+    @notice Called during vault and strategy reports 
+    to retreive the protocol fee to charge and address
+    to receive the fees.
+    @return The protocol fee config for the msg sender
+    """
+    # If there is a custom protocol fee set we return it.
+    if self.use_custom_protocol_fee[msg.sender]:
+        # Always use the default fee recipient even with custom fees.
+        return PFConfig({
+            fee_bps: self.custom_protocol_fee[msg.sender],
+            fee_recipient: self.default_protocol_fee_config.fee_recipient
```

Comment:
The problem with saving the fee_recipient in the same struct as the custom fee is that the address would not be updated for custom fee vaults when it is updated for the normal config.

Gov would then have to track every custom fee set previous to any fee_recipient updates and manually change each one. Which potentially could be a huge amount.

Though this should be rare, it seems worth the extra sload to never worry about that.

Reply:
That is what is happening with the accountant. 

Though not sure we want the factory to be holding a bunch of funds from hundreds of vaults/strategies. And probably want to keep the contract as simple as possible.

Having the address updateable also means multiple versions all route their protocol fees to the same treausry, rather than having them all be in seperate contracts that need to be pulled